Sources/PointFree/Session.swift

```diff
@@ -13,6 +14,18 @@ public enum CookieTransform: String, Codable {
   case encrypted
 }
 
+extension CookieTransform: DependencyKey {
+  public static let liveValue = Self.encrypted
+  public static let testValue = Self.plaintext
```

Comment:
fun example of a dependency that doesn't need an "unimplemented" test value. best to just use plaintext cookies.